man/coredump.conf.xml

```diff
@@ -97,11 +97,14 @@
         <term><varname>ExternalSizeMax=</varname></term>
         <term><varname>JournalSizeMax=</varname></term>
 
-        <listitem><para>The maximum (compressed or uncompressed) size in bytes of a core to be saved in
-        separate files on disk (default: 1G on 32bit, 32G on 64bit systems) or in the journal (default:
-        10M). Unit suffixes are allowed just as in <option>ProcessSizeMax=</option>.</para></listitem>
-
-        <para><varname>ExternalSizeMax=infinity</varname> sets the core size to unlimited.</para>
+        <listitem><para>The maximum (compressed or uncompressed) size in bytes of a coredump to be saved in
+        separate files on disk (default: 1G on 32bit systems, 32G on 64bit systems) or in the journal
+        (default: 767M). Note that the journal service enforces a hard limit on journal log records of 767M,
+        and will ignore larger submitted log records. Hence, <varname>JournalSizeMax=</varname> may be
+        lowered relative to the default, but not increased. Unit suffixes are allowed just as in
+        <option>ProcessSizeMax=</option>.</para>
```

Comment:
Pre-existing condition, but maybe say "767 MiB", "1 GiB", etc, and '32-bit', '64-bit'.

Reply:
i made the 32-bit + 64bit-change now. I didn't do the MiB/GiB change though, because I think there's value here denoting this in a way that we actually parse ourselves as valid size. And the thing is that parse_size() cannot parse "MiB" expressions. I think it would be really weird documenting a default here that you cannot actually specify validly.

Hope that makes sense?

Reply:
Yeah. I wonder if we should start accepting MiB and GiB… But that'd be a larger project.